
APR and APR-util are patched to store dependencies in the .la file.

--- subversion-0.20.1/build/ac-macros/aprutil.m4.deplibs	2003-03-26 16:34:47.000000000 +0000
+++ subversion-0.20.1/build/ac-macros/aprutil.m4	2003-04-29 11:03:54.000000000 +0100
@@ -64,11 +64,9 @@
     AC_MSG_ERROR([apu-config --includes failed])
   fi
 
-  dnl When APR stores the dependent libs in the .la file, we don't need
-  dnl --libs.
-  SVN_APRUTIL_LIBS="`$apu_config --link-libtool --libs`"
+  SVN_APRUTIL_LIBS="`$apu_config --link-libtool`"
   if test $? -ne 0; then
-    AC_MSG_ERROR([apu-config --link-libtool --libs failed])
+    AC_MSG_ERROR([apu-config --link-libtool failed])
   fi
 
   SVN_APRUTIL_EXPORT_LIBS="`$apu_config --link-ld --libs`"
--- subversion-0.20.1/build/ac-macros/apr.m4.deplibs	2003-03-26 16:34:47.000000000 +0000
+++ subversion-0.20.1/build/ac-macros/apr.m4	2003-04-29 11:03:45.000000000 +0100
@@ -68,9 +68,9 @@
 
   dnl When APR stores the dependent libs in the .la file, we don't need 
   dnl --libs.
-  SVN_APR_LIBS="`$apr_config --link-libtool --libs`"
+  SVN_APR_LIBS="`$apr_config --link-libtool`"
   if test $? -ne 0; then
-    AC_MSG_ERROR([apr-config --link-libtool --libs failed])
+    AC_MSG_ERROR([apr-config --link-libtool failed])
   fi
 
   SVN_APR_EXPORT_LIBS="`$apr_config --link-ld --libs`"
